qiita_pet/handlers/user_handlers.py

```diff
@@ -185,3 +187,11 @@ def post(self):
 
         self.render("user_messages.html",
                     messages=self.current_user.messages())
+
+
+class UserJobs(BaseHandler):
+    @authenticated
+    @coroutine
+    def get(self):
+        response = user_jobs_get_req(self.current_user)
```

Comment:
I believe this needs take the result of a `yield`? 

http://www.tornadoweb.org/en/stable/gen.html